api/src/main/java/com/codeforcommunity/dto/school/SchoolListResponse.java

```diff
@@ -5,18 +5,18 @@
 public class SchoolListResponse {
 
   private int count;
-  private List<School> schools;
+  private List<SchoolSummary> schools;
 
-  public SchoolListResponse(List<School> schools, int count) {
+  public SchoolListResponse(List<SchoolSummary> schools, int count) {
     this.schools = schools;
     this.count = count;
   }
 
-  public List<School> getSchools() {
+  public List<SchoolSummary> getSchools() {
     return schools;
```

Comment:
Should we return a copy of the schools to avoid any other class mutating this object's list of schools?